extensions/google-drive/src/utils.ts

```diff
@@ -161,6 +164,33 @@ ${new Date(file.createdAt).toLocaleString()}
 ${new Date(file.updatedAt).toLocaleString()}`;
 };
 
+export const fileStream = ({ stats = false }: { stats?: boolean } = {}) => {
+  const driveRootPath = getDriveRootPath();
+  const preferences = getPreferenceValues<Preferences>();
+
+  const filter = new Transform({
+    objectMode: true,
+    transform(chunk: Entry, encoding, callback) {
+      // Don't include directories if the preference is false
+      callback(null, chunk.dirent.isDirectory() && !preferences.shouldShowDirectories ? undefined : chunk);
+    },
+  });
+
+  const excludePaths = getExcludePaths()
+    .concat(IGNORED_DIRECTORIES)
+    .map((p) => path.join("**", p));
```

Comment:
We don't want globbing syntax for the `excludePaths` preference (since they'd be absolute paths.

It is fine for the `IGNORED_DIRECTORIES`.